src/BloomBrowserUI/bookEdit/StyleEditor/StyleEditor.ts

```diff
@@ -534,6 +534,11 @@ class StyleEditor {
 
         $('#format-toolbar').remove(); // in case there's still one somewhere else
 
+        // BL-2476: Readers made from BloomPacks should have the formatting dialog disabled
+        var suppress = $(document).find('meta[name="suppressFormatting"]');
```

Comment:
Please check the issue comments. I think JohnH requested that we call this 'lockFormatting' rather than 'suppressFormatting'